Sunday Default.xml

```diff
@@ -749,42 +749,26 @@
             <Op id="PreviewInput" mixEffectBlockIndex="1" input="Camera20"/>
         </Macro>
         <Macro index="15" name="Ben Third" description="">
-            <!-- Make sure Key 3 is off so we can transition it on -->
-            <Op id="KeyOnAir" mixEffectBlockIndex="0" keyIndex="2" onAir="False"/>
-            <!-- Set Ben Animated Lower Third Clip -->
-            <Op id="HyperDeckSetSourceClipIndex" hyperDeckIndex="3" clipIndex="2"/>
+            <!-- Set Will Animated Lower Third Clip -->
+            <Op id="HyperDeckSetSourceClipIndex" hyperDeckIndex="3" clipIndex="3"/>
```

Comment:
This happened again. Check the comment line above this one too.

Reply:
Damn it.